test/unit/test_sap_cli_aunit.py

```diff
@@ -281,6 +282,30 @@ def test_aunit_class_junit4_syntax_error(self):
 '''''')
         self.assertEqual(retval, 1)
 
+    def test_aunit_class_junit4_skipped(self):
+        self.connection.set_responses(Response(status_code=200, text=AUNIT_RESULTS_SKIPPED_XML, headers={}))
+
+        with patch('sap.cli.core.get_console', return_value=BufferConsole()) as mock_print:
+            retval = self.execute_run('class', 'yclass', '--output', 'junit4', '--result', ResultOptions.ONLY_UNIT.value)
+
+        self.maxDiff = None
+        self.assertEqual(mock_print.return_value.capout, '''<?xml version="1.0" encoding="UTF-8" ?>
+<testsuites name="yclass">
+  <testsuite name="LTCL_TEST" package="ZCL_THEKING_MANUAL_HARDCORE" tests="1">
+    <testcase name="DO_THE_FAIL" classname="ZCL_THEKING_MANUAL_HARDCORE=&gt;LTCL_TEST" status="SKIP">
```

Comment:
Does it make sense to keep `status` attribute in xml output?

Reply:
The attribute `status` is not banned and it's already there and perhaps it works for somebody.